tests/accessor/accessor_default_values_fp64.cpp

```diff
@@ -0,0 +1,45 @@
+/*******************************************************************************
+//
+//  SYCL 2020 Conformance Test Suite
+//
+//  Provides tests for accessor default values.
+//
+//  This test provides verifications that template parameters has default values
+//  for generic accessor, host_accessor and local_accessor with double type.
+//
+*******************************************************************************/
+
+#include "../common/common.h"
+#include "../common/disabled_for_test_case.h"
+
+// FIXME: re-enable when sycl::accessor is implemented
+#if !defined(__HIPSYCL__) && !defined(__COMPUTECPP__) && \
+    !defined(__SYCL_COMPILER_VERSION)
```

Comment:
```suggestion
#if !SYCL_CTS_COMPILING_WITH_HIPSYCL && !SYCL_CTS_COMPILING_WITH_COMPUTECPP && \
    !SYCL_CTS_COMPILING_WITH_DPCPP
```

Reply:
I've applied this comment in this commit: https://github.com/KhronosGroup/SYCL-CTS/pull/335/commits/66c0562df41dca778c91018c6669d171f18b0807